examples/user_interfaces/embedding_in_wx3.py

```diff
@@ -134,19 +137,22 @@ def OnInit(self):
         plot_container.SetSizer(sizer)
 
         # whiz button ------------------
-
         whiz_button = xrc.XRCCTRL(self.frame, "whiz_button")
-        wx.EVT_BUTTON(whiz_button, whiz_button.GetId(),
-                      self.plotpanel.OnWhiz)
+        if 'phoenix' in wx.PlatformInfo:
+            whiz_button.Bind(wx.EVT_BUTTON, self.plotpanel.OnWhiz)
+        else:
+            wx.EVT_BUTTON(whiz_button, whiz_button.GetId(),
+                          self.plotpanel.OnWhiz)
```

Comment:
I don't have anything older then 2.8.12.1 to test against, in that version Bind() is fine.  Currently mpl states that minimum is 2.8, do we change that?